docs/product-migration-guide/src/main/asciidoc/migration-configure-kie-server-proc.adoc

```diff
@@ -0,0 +1,41 @@
+[id='migration-configure-kie-server-proc']
+= Configure and start {KIE_SERVER}
+
+After you have migrated all {CENTRAL} data and have upgraded the relevant database, configure and start the new {KIE_SERVER} to transfer the execution server functionality and apply the migrated changes.
+
+.Prerequisites
+* Project data from {PRODUCT_OLD} 6.4 has been migrated using the {CENTRAL} migration tool.
+* The {PRODUCT_OLD} 6.4 database to be used with {PRODUCT} {PRODUCT_VERSION} has been upgraded.
+* All API references to {CENTRAL} execution server in application code have been redirected to the {KIE_SERVER}.
+
+.Procedure
+Navigate to the {PRODUCT} {PRODUCT_VERSION} `bin` directory and start the new {KIE_SERVER} with the following properties. Adjust the specific properties according to your environment.
+
+[source]
+----
+$ ~/EAP_HOME/bin/standalone.sh - c server-config=standalone-full.xml <1>
+-Dorg.kie.server.id=myserver <2>
+-Dorg.kie.server.controller=http://localhost:8080/business-central/rest/controller <3>
+-Dorg.kie.server.persistence.dialect=org.hibernate.dialect.PostgreSQLDialect <4>
+-Dorg.kie.server.persistence.ds=java:jboss/datasources/psjbpmDS <5>
+----
+<1> Start command with `standalone-full.xml` server profile
+<2> Server ID that must match the server template name defined in {CENTRAL}
+<3> Controller location, {CENTRAL} URL with `/rest/controller` suffix
+<4> Hibernate dialect to be used
+<5> JNDI name of the data source used for your previous {PRODUCT_OLD} database
+
+[NOTE]
+====
+If {CENTRAL} and {KIE_SERVER} are deployed with separate application server instances ({EAP} or other), then also include a port offset and the offset {KIE_SERVER} location in the {KIE_SERVER} properties to avoid port conflicts:
```

Comment:
AFAIK port offset is just a quick way to run EAP on a different port than 8080, so if a user has it configured elsewhere, port offset is not needed. Maybe I would tell here: "then you may want to include a port offset... to avoid port conflicts if the EAP is not configured to run on a different port than the default one."

Reply:
Good call. Changed to: "If Business Central and Process Server are deployed with separate application server instances (Red Hat JBoss EAP or other), then include also the separate location of Process Server to avoid port conflicts with Business Central. If a separate Process Server port has not been configured, you can add a port offset and the offset Process Server location in the Process Server properties."